_data/banking.yml

```
@@ -433,6 +433,14 @@ websites:
       img: fifththird.png
       tfa: No
 
+    - name: Fio.cz
```

On both Twitter and Facebook they refer to themselves as "Fio banka" so I suggest you change the name tag to reflect that.
```suggestion
    - name: Fio banka
```